services/report_status.go

```diff
@@ -1,15 +1,38 @@
-package boyar
+package services
 
 import (
 	"context"
 	"fmt"
 	"github.com/orbs-network/boyarin/log_types"
+	"github.com/orbs-network/boyarin/strelets"
 	"github.com/orbs-network/boyarin/strelets/adapter"
+	"github.com/orbs-network/boyarin/supervized"
 	"github.com/orbs-network/scribe/log"
 	"time"
 )
 
-func ReportStatus(ctx context.Context, logger log.Logger, since time.Duration) error {
+const SERVICE_STATUS_REPORT_PERIOD = 1 * time.Minute
+const SERVICE_STATUS_REPORT_TIMEOUT = 30 * time.Second
+
+func WatchAndReportServicesStatus(logger log.Logger) {
```

Comment:
You can pass the list of services and get rid of `GetVcidFromServiceName` altogether.